Engine/Helper.cs

```diff
@@ -309,7 +311,7 @@ public PSModuleInfo GetModuleManifest(string filePath, out IEnumerable<ErrorReco
             {
                 using (var ps = System.Management.Automation.PowerShell.Create())
                 {
-                    ps.RunspacePool = _runSpacePool;
+                    ps.RunspacePool = _runspacePool;
```

Comment:
Given that this is locked above, is it worthwhile having a runspace pool here at all? Two solutions would be:

- Allow `ps` to derive its own runspace
- Have a private `Lazy<Runspace>` field that only this method reuses

Reply:
I like the former - I'll just eliminate the runspace pool in the Helper, I feel it's the cache where we really need it